package-lock.json

```diff
@@ -1,8 +1,8584 @@
 {
   "name": "@codewars/discord-bot",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
What version of Node.js did you use? I'd rather not have the lockfile updated like this in this PR.